controller/eks-cluster-config-handler.go

```diff
@@ -353,11 +353,16 @@ func validateUpdate(config *eksv1.EKSClusterConfig) error {
 		if clusterVersion.EQ(*version) {
 			continue
 		}
-		if clusterVersion.Minor-version.Minor == 1 {
+		if (clusterVersion.Minor < 25 && clusterVersion.Minor-version.Minor <= 2) || (clusterVersion.Minor >= 25 && clusterVersion.Minor-version.Minor <= 3) {
```

Comment:
Why it has to be lower than 25? We already dropped support for Kubernetes lower than 27. It doesn't make sense to me